app/models/hmrc/response.rb

```diff
@@ -5,5 +5,9 @@ class Response < ApplicationRecord
     USE_CASES = %w[one two].freeze
     belongs_to :legal_aid_application, inverse_of: :hmrc_responses
     validates :use_case, presence: true, inclusion: { in: USE_CASES, message: 'Invalid use case' }
+
+    def self.use_case_one_for(laa_id)
```

Comment:
Should there be a `use_case_two_for` as well?

Reply:
I think I get everything I need from use_case_one (date, gross pay, tax, national insurance), so I don't think we need it

Reply:
I had a vague recollection of DFab saying one filed was needed from UC2, but fair enough, we can always amend if needed :+1: